charts/sck-otel/ci/filled-values.yaml

```diff
@@ -243,32 +246,26 @@ agent:
   ports:
     otlp:
       containerPort: 4317
-      hostPort: 4317
       protocol: TCP
       enabled:  # true for splunkObservability
     sfx-forwarder:
       containerPort: 9080
-      hostPort: 9080
       protocol: TCP
       enabled:  # true for splunkObservability and if tracesEnabled=true
     zipkin:
       containerPort: 9411
-      hostPort: 9411
       protocol: TCP
       enabled:  # true for splunkObservability and if tracesEnabled=true
     jaeger-thrift:
       containerPort: 14268
-      hostPort: 14268
       protocol: TCP
       enabled:  # true for splunkObservability and if tracesEnabled=true
     jaeger-grpc:
       containerPort: 14250
-      hostPort: 14250
       protocol: TCP
       enabled:  # true for splunkObservability and if tracesEnabled=true
     signalfx:
       containerPort: 9943
-      hostPort: 9943
```

Comment:
Why removed? host ports are required in the agent mode to receive traces from local pods

Reply:
this is only for chart testing in github actions